desktop/cmp/grid/editors/impl/InlineEditorModel.js

```diff
@@ -47,7 +46,7 @@ export function useInlineEditorModel(component, props, ref, isPopup = false) {
         model: impl,
         bind: 'value',
         commitOnChange: true,
-        ref: composeRefs(ref, impl.ref),
+        ref: impl.ref,
```

Comment:
this seems correct to me -- surprised it was not causing us even more problems before